platform_plugin_communications/urls.py

```diff
@@ -1,10 +1,14 @@
 """
 URLs for platform_plugin_communications.
 """
-from django.urls import re_path  # pylint: disable=unused-import
-from django.views.generic import TemplateView  # pylint: disable=unused-import
+from django.urls import path
+
+from platform_plugin_communications.api.views import send_email_api_view
 
 urlpatterns = [
-    # TODO: Fill in URL patterns and views here.
-    # re_path(r'', TemplateView.as_view(template_name="platform_plugin_communications/base.html")),
+    path(
+        "send_email_to_learners",
+        send_email_api_view,
+        name="send_email_to_individual_learners",
```

Comment:
if we're using the same path for teams/individual learners, can we update this to be more generic?